app/src/main/java/org/fossasia/pslab/fragment/ApplicationsFragment.java

```diff
@@ -40,18 +45,36 @@ public static ApplicationsFragment newInstance() {
     public View onCreateView(final LayoutInflater inflater, @Nullable ViewGroup container,
                              @Nullable Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.applications_fragment, container, false);
-        // Initiate context
         context = getActivity().getApplicationContext();
-        // Initiate resources
         applicationItemList = new ArrayList<>();
         applicationAdapter = new ApplicationAdapter(context, applicationItemList,
                 new ApplicationAdapter.OnItemClickListener() {
-            @Override
-            public void onItemClick(ApplicationItem item) {
-                Toast.makeText(context, item.getApplicationName(),
-                        Toast.LENGTH_LONG).show();
-            }
-        });
+                    @Override
+                    public void onItemClick(ApplicationItem item) {
+                        Toast.makeText(context, item.getApplicationName(),
+                                Toast.LENGTH_LONG).show();
```

Comment:
Yeah, but I kept them as I haven't added classes for some applications like sensor  quick view, data sensor logger, etc. So kept it as an indication that they have been clicked.